bindings/cs/MongoDB.Libmongocrypt/BinarySafeHandle.cs

```diff
@@ -51,7 +51,7 @@ public override bool IsInvalid
             }
         }
 
-#if NET452 || NETSTANDARD2_0
+#if NET472 || NETSTANDARD2_0
```

Comment:
`#if` not required anymore as `ReliabilityContractd` is present on all supported platforms.